specs/Transportation/TrafficFlowObserved/schema.json

```diff
@@ -74,6 +74,9 @@
             "backward"
           ]
         },
+        "vehicleType": {
```

Comment:
you need to convey what the possible vehicle types are. Please check the VehicleMode schema for a list of values. 

In addition you would need to modify the spec.md file to reflect the new attribute

thanks

Reply:
Hi, I have been looking into the vehicle types that are provided under the Vehicle Model, where it says the allowed values are defined by VehicleTypeEnum and VehicleTypeEnum2, DATEX 2 version 2.3

the allowed values ( agriculturalVehicle, bicycle, bus, minibus, car, caravan, tram, tanker, carWithCaravan, carWithTrailer, lorry, moped, tanker, motorcycle, motorcycleWithSideCar, motorscooter, trailer, van, caravan, constructionOrMaintenanceVehicle) do not necessarily easily fit into our vehicle classification ( LGV, OGV1, OGV2, car, etc). How are the allowed values maintained or changed, and how would I go about getting some values added?

Reply:
just to let you know, I have contacted synchronicity and they have answered the above question for me, I will look to updating the model and issuing a new pull request